content/01.main_content.md

```diff
@@ -1,3 +1,15 @@
+Recognition by peers or the general public can greatly affect the trajectory of a scientist’s career.
+  Whether the recognition is the inclusion of their name in print as an expert, an invited lecture, or an award, each form of recognition paves the way to future accolades.
+  Regardless of an individual’s merit, biases can skew which scientists are even considered for recognition.
+  To combat biases, auditing systems can identify if specific groups are being unintentionally neglected and hold the recognizing body accountable.
```

Comment:
To be clear, is "auditing" part of the noun, a la "dressing gown", or is it a verb? Are we proposing creating something known as "auditing systems" or are we proposing that we audit the existing systems? Don't know if that makes sense but will help my understanding of the rest of the paper.

Reply:
aha, yes like dressing gown. I'll replace with "audits"

Reply:
edited in https://github.com/greenelab/disparities_opinion/pull/3/commits/7eca82b37f14dfa33ed8974e10d3d8cc2f9931c1